packages/core-ethereum/package.json

```diff
@@ -27,7 +27,6 @@
     "lib"
   ],
   "dependencies": {
-    "@hoprnet/hopr-ethereum": "workspace:packages/ethereum",
```

Comment:
Why are we removing this dependency?

Reply:
Because `ethereum` does not publish itself as an npm package. `core-ethereum` doesn't use anything from `hopr-ethereum`